templates/confidential-computing/index.html

```diff
@@ -56,163 +76,124 @@ <h2>Benefits of Ubuntu for confidential computing</h2>
+
+        {%- if slot == 'list_item_description_3' -%}
+          <p>We support a comprehensive package, featuring a 6.8 kernel, Shim, Grub, and TDVF, which serves as an in-guest VM firmware.</p>
+        {%- endif -%}
+
+        {%- if slot == 'cta' -%}
+          <a href="https://github.com/canonical/tdx">Deploy Intel TDX with Ubuntu 24.04 today&nbsp;&rsaquo;</a>
+        {%- endif -%}
+      {%- endcall -%}
     </section>
 
     <section class="p-section">
-      <hr class="p-rule--muted is-fixed-width" />
+      <hr class="p-rule is-fixed-width" />
       <div class="row--50-50">
         <div class="col">
           <h2>How confidential VMs work</h2>
         </div>
         <div class="col">
-          <p>
-            Confidential VMs introduce a new trust boundary which only includes
-            the software running within, and the platform’s hardware. All other
-            software outside is no longer part of your trusted computing base.
-          </p>
-          <div class="p-image__background--3-2 u-align--center u-vertically-center">
-            {{ image(url="https://assets.ubuntu.com/v1/abd7bccf-Hypervisor%20diagram%203.png",
-                        alt="Diagram showing the hardware is at the base and the CVM can add a layer of protection around the software
-                        it's running",
-                        width="195",
-                        height="282",
-                        hi_def=True,
-                        loading="lazy") | safe
-            }}
+          <div class="row u-sv3">
+            <p>
+              Confidential VMs introduce a new trust boundary which only includes the software running within, and the platform’s hardware. All other software outside is no longer part of your trusted computing base. To provide such strong security guarantees, confidential computing relies on two main primitives:
+            </p>
+            <div class="p-image-container is-highlighted u-sv3">
+              {{ image(url="https://assets.ubuntu.com/v1/480ebd6e-how-confidential-vms.png",
+                alt="",
+                width="1200",
+                height="800",
+                hi_def=True,
+                loading="lazy"
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="row">
+            <ol class="p-stepped-list">
+              <li class="p-stepped-list__item">
+                <h5 class="p-stepped-list__title">
+                  Isolation
+                </h5>
+                <p class="p-stepped-list__content">Confidential computing capable CPUs are equipped with an AES hardware memory encryption engine, which encrypts data when it is written to system memory, and decrypts it when read. The encryption key itself is stored in the hardware root of trust and is never exposed to the platform’s system software.</p>
+              </li>
+              <hr class="p-rule--muted" />
+              <li class="p-stepped-list__item">
```

Comment:
This should be moved inside of the list so that the ol doesn't have any invalid direct children
```suggestion
              <li class="p-stepped-list__item">
                <hr class="p-rule--muted" />
```